src/js/jsx/sections/style/Type.jsx

```diff
@@ -506,26 +517,53 @@ define(function (require, exports, module) {
                     fontWeight: this._getCSSFontWeight(styleTitle) || 400
                 };
 
-                if (colorTiny) {
-                    typeStyle.color = colorTiny.toRgbString();
-                }
+                if (familyName === strings.STYLE.TYPE.MIXED) {
+                    var fillStyle = {
+                        height: "100%",
+                        width: "100%",
+                        backgroundColor: colorTiny ? colorTiny.toRgbString() : "transparent"
+                    };
 
-                return (
-                    <div
-                        className="type__preview"
-                        style={typeStyle}>
-                        Aa
-                    </div>
-                );
+                    return (
+                        <div
+                            className="fill__preview"
+                            style={fillStyle}/>
+                    );
+                } else {
+                    typeStyle = {
+                        fontFamily: familyName || "helvetica",
+                        fontStyle: this._getCSSFontStyle(styleTitle) || "regular",
+                        fontWeight: this._getCSSFontWeight(styleTitle) || 400,
+                        fontSize: 24
+                    };
+
+                    if (colorTiny) {
+                        typeStyle.color = colorTiny.toRgbString();
+                    }
+
+                    return (
+                        <div
+                            className="type__preview"
+                            style={typeStyle}>
+                            Aa
+                        </div>
+                    );
+                }
             }.bind(this);
 
             var colorPickerID = "type-" + this.props.document.id,
                 typefaceListID = "typefaces-" + this.props.document.id,
-                weightListID = "weights-" + this.props.document.id;
-
-            return (
-                <div ref="type">
-                    <div className="formline">
+                weightListID = "weights-" + this.props.document.id,
+                fillClassNames = classnames("formline", {
+                    "mixed-faces": familyName === strings.STYLE.TYPE.MIXED
+                }),
+                fillBlendFormline;
+            
+            if (this.props.mixedVectorType) {
+                fillBlendFormline = null;
+            } else {
```

Comment:
Assigning `fillBlendFormline` to null seems not necessary here, as the default value `undefined` will do the job.